modules/RTC/TraceablePeerConnection.js

```diff
@@ -1101,16 +1104,14 @@ TraceablePeerConnection.prototype._remoteTrackRemoved = function(stream, track)
     }
 
     const toBeRemoved = this.getRemoteTracks().find(
-        remoteTrack => remoteTrack.getStreamId() === streamId
-        && remoteTrack.getTrackId() === trackId);
+        remoteTrack => remoteTrack.getStreamId() === streamId && remoteTrack.getTrackId() === trackId);
 
     if (!toBeRemoved) {
         GlobalOnErrorHandler.callErrorHandler(new Error(`${this} remote track removal failed - track not found`));
 
         return;
     }
 
-    logger.info(`${this} remote track removed stream[id=${streamId},trackId=${trackId}]`);
```

Comment:
Why we are removing this log? I've found it quite useful for debugging in some cases.